lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -696,3 +696,41 @@ function dosomething_helpers_get_campaign_status($start_date, $end_date) {
     return 'closed';
   }
 }
+
+/**
+ * Gets the current run for a given campaign nid and user.
+ *
+ * @param int $nid
+ * @param object $user
+ *  Optional. Will default to global $user if not supplied
+ *
+ * @return node $run
+ *  Returns the campaigns node in the correct language.
+*/
+function dosomething_helpers_get_campaign_run($nid, $user = NULL) {
```

Comment:
I updated this function to use the new `field_current_run` but it looks like @deadlybutter's function takes into account business logic differently. Also it returns the whole run node, not just the nid

https://github.com/DoSomething/phoenix/blob/dev/lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.module#L224